test/models/player_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class PlayerTest < ActiveSupport::TestCase
+  let(:player) do
+    Player.new(
+      game: game,
+      name: "Matt",
+      seat_number: 1,
+      faction: Faction.find_by_key(faction_key),
+    )
+  end
+  let(:faction_key) { "l1z1x" }
+  let(:game) { games(:one) }
+
+  it "creates a valid player" do
+    _(player.valid?).must_equal true
```

Comment:
To be honest, I had to look this up :) Haven't used `Minitest::Spec` before (but looking forward in doing so) and this really surprised me ^^ I have at least some concerns about this particular style:

1. It conflicts with the magic `_` method which references the last return value
2. It makes searching for it quite hard :)

Would you be very attached to this? Or could we use `expect` for my "aesthetic pleasure" (as said by the authors ^^)

Reply:
> Would you be very attached to this? Or could we use expect for my "aesthetic pleasure" (as said by the authors ^^)

No, I am absolutely not attached. I have not yet written a lot of tests with MiniTest and was following examples documented in the README of [`minitest-spec-rails`](https://github.com/metaskills/minitest-spec-rails#test-styles) which caused a deprecation notice:

> DEPRECATED: global use of must_equal from test/models/player_test.rb:24. Use _(obj).must_equal instead. This will fail in Minitest 6.

So I went with this `_()` syntax and was not even aware that `expect()` is an alternative :)

I swapped `_()`-calls with `expect()`